apps/services/party-letter-registry-api/migrations/20210514135724-party-letter-registry.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const transaction = await queryInterface.sequelize.transaction()
+
+    // try is here to make sure this safely fails if index creation fails
+    try {
+      await queryInterface.createTable('party_letter_registry', {
+        id: {
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          primaryKey: true,
+        },
+        party_letter: {
+          type: Sequelize.CHAR(2),
+          allowNull: false,
+          unique: true,
+        },
+        party_name: {
+          type: Sequelize.STRING,
+          allowNull: false,
+        },
+        owner: {
```

Comment:
Is owner a single person? And is it not possible for a single person to be the owner of more than one party?

Reply:
Owner is a single person or company.
Single national id per party letter covers the required conditions for our current project.
It might be theoretically possible to have more than on political party assigned to a single national id